pkg/services/accesscontrol/database/database.go

```diff
@@ -16,8 +16,10 @@ import (
 // TimeNow makes it possible to test usage of time
 var TimeNow = time.Now
 
+// Due to the way we organize code, without having AccessControlStore as a service it is hard/not possible to run migrations when writing tests.
+// SQLStore is not injected directly to make dependency between AccessControlStore and Manager explicit.
 type AccessControlStore struct {
-	SQLStore *sqlstore.SQLStore `inject:""`
+	SQLStore *sqlstore.SQLStore
```

Comment:
Dependencies are a bit hard to manage here. While initially I was hoping that I could skip registering access control store as a service, but I run into too many problems with tests, when running migrations. I eventually decided to not inject the SQLStore and init it from the manager, but keep the accesscontrolstore as a service.